Library/Formula/tvnamer.rb

```diff
@@ -0,0 +1,35 @@
+class Tvnamer < Formula
+  homepage "https://github.com/dbr/tvnamer"
```

Comment:
Could you add a short description? The syntax is `desc "the description…"`. Thanks!